src/Lighthouse/LighthouseHostFactory.cs

```diff
@@ -45,7 +45,7 @@ public static class LighthouseHostFactory
             if (lighthouseConfig != null && IsNullOrEmpty(systemName))
                 systemName = lighthouseConfig.GetString("actorsystem", systemName);
 
-            ipAddress = clusterConfig.GetString("akka.remote.dot-netty.tcp.public-hostname");
+            ipAddress = clusterConfig.GetString("akka.remote.dot-netty.tcp.public-hostname", "127.0.0.1");

```

Comment:
When trying to start Lighthouse without environment configuration provided, it does not find `public-hostname` setting in hocon and throws NRE later (when trying to make `ipAddress.Trim()`).
I think warnings about issues connecting to `127.0.0.1` is more informative, and at least does not look so scary :) Maybe even better would be to add some `log.warning` about missing public hostname here?

Reply:
I think that's a good idea - we should add it.